src/store/memberStore.js

```diff
@@ -0,0 +1,10 @@
+import create from 'zustand';
+
+const useMemberStore = create((set) => ({
+  members: [],
+  showMembers: false,
+  showMembersHandler: (filter) => set(() => ({ showMembers: filter })),
```

Comment:
As we are using this function for toggling the state only, can we do it over here? Something like `toggleShowMembers` so that you can remove functions where we have set it to be `false` or `true` and just use toggle? What do you think?